lib/reporter.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+function installCommand(type) {
+  switch (type) {
+  case 'npm':
+    return 'npm install';
+  case 'npm-shrinkwrap':
+    return 'rm -rf node_modules/ && npm install';
```

Comment:
I am not sure about this, but isn't it better to just uninstall the problematic dependency and install it again.